cmake/kokkos_arch.cmake

```diff
@@ -1044,6 +1044,12 @@ FOREACH(ARCH IN LISTS SUPPORTED_AMD_ARCHS)
       SET(KOKKOS_ARCH_NAVI1100 ON)
       BREAK()
     ENDIF()
+    STRING(REGEX MATCH "1103" IS_1103 ${ARCH})
+    IF(IS_1103)
+      SET(KOKKOS_ARCH_AMD_GFX1103 ON)
+      SET(KOKKOS_ARCH_NAVI ON)
+      BREAK()
+    ENDIF()
```

Comment:
This was done for backward compatibility. This is not necessary when adding a new backend.

Reply:
it is gone now.